maquetado/scss/_titles.scss

```diff
@@ -0,0 +1,10 @@
+.title {
+  background: transparent url('assets/triangle.svg') no-repeat center left;
+  color: $titles-color;
```

Comment:
This color name is wrong. This name should describe how the color looks and not where it is used. You could read [this](https://github.com/Wolox/tech-guides/blob/master/frontend/docs/css-style-guide.md#colors)